packages/yoga/src/Menu/web/MenuItem.jsx

```diff
@@ -50,7 +52,7 @@ const StyledMenuItem = styled.li`
 
     svg {
       margin-right: ${menu.margin.default}px;
-      fill: ${menu.icon.disabled};
+      fill: ${iconColor};
```

Comment:
I was thinking here and would be good to set the yoga token as the colors, the prop keeps, but using the same idea as the `size`.